client/src/cb-admin/pages/__tests__/ConfirmPassword.test.js

```diff
@@ -14,17 +14,21 @@ import ConfirmPassword from '../ConfirmPassword';
 describe('ConfirmPassword Component', () => {
   let mock;
 
-  beforeAll(() => {
-    mock = new MockAdapter(axios);
-  });
+  if (!process.env.LIVE_API) {
```

Comment:
If this goes ahead, we'll be doing this quite a lot, so it's worth coming up with a way to accommodate it without having to stick `if` statements all over the place. Maybe something like a function that constructs a new `MockAdapter` instance if `LIVE_API` is `true`, otherwise an object with the same API, whose methods all do nothing?